packages/pagebuilder/lib/factory.js

```diff
@@ -1,5 +1,26 @@
 import React, { Suspense } from 'react';
-import { getContentTypeConfig } from './config';
+import { getContentTypeConfig, setContentTypeConfig } from './config';
+import customContentTypes from './ContentTypes/customContentTypes';
+
+/**
+ *  add custom content types
+ */
+const addCustomContentTypes = contentTypes => {
```

Comment:
Pls, add test coverage to: 
https://github.com/magento/pwa-studio/blob/1977f38305ff6c0e2b23a9da7beb0b2f69758bed/packages/pagebuilder/lib/__tests__/factory.spec.js

Reply:
I didn't export this function before, should I do it now for the test?

Reply:
sorry this is true :smile_cat: I am fine with it don't make sense to export to get coverage here